src/Sylius/Bundle/ProductBundle/spec/Form/Type/ProductTypeSpec.php

```diff
@@ -57,6 +62,21 @@ function it_builds_form_with_proper_fields(FormBuilder $builder)
             ->willReturn($builder)
         ;
 
+        $builder
```

Comment:
It's already covered by code at line 42. There's no need for duplicating it, if there's no `shouldBeCalled()`.